helm/nessie/values.yaml

```diff
@@ -72,17 +72,18 @@ mongodb:
     # -- The secret key storing the MongoDB password.
     password: mongodb_password
 
-# Postgres settings. Only required when using JDBC version store type; ignored otherwise.
-postgres:
-  # -- The Postgres JDBC connection string.
+# JDBC datasource settings. Only required when using JDBC version store type; ignored otherwise.
+jdbc:
```

Comment:
Same for the secrets-stuff below

Reply:
It isn't: the old `postgres` section is still honored thanks to the "helm magic" ;-)

IOW you can do this:

```shell
 helm install nessie -n nessie-ns helm/nessie --debug --dry-run \
  --set versionStoreType=JDBC  \
  --set postgres.jdbcUrl=jdbc:postgresql://foo/bar \
  --set postgres.secret.password=old-password-key
```

And it will merge the old `postgres.xyz` values with the new defaults coming from `jdbc.xyz`:

```yaml
- name: NESSIE_VERSION_STORE_PERSIST_JDBC_DATASOURCE
  value: postgresql
- name: QUARKUS_DATASOURCE_POSTGRESQL_USERNAME
  valueFrom:
    secretKeyRef:
      name: datasource-creds
      key: username
- name: QUARKUS_DATASOURCE_POSTGRESQL_PASSWORD
  valueFrom:
    secretKeyRef:
      name: datasource-creds
      key: old-password-key
- name: QUARKUS_DATASOURCE_POSTGRESQL_JDBC_URL
  value: jdbc:postgresql://foo/bar
```